src/Google.Api.Gax/Platform.cs

```diff
@@ -426,6 +442,8 @@ public GkePlatformDetails(string metadataJson, string projectId, string clusterN
 
         /// <summary>
         /// The cluster location, which is visible in the Google Cloud Platform Console.
+        /// This is equivalent to the value of the <code>&lt;zone-name&gt;</code> part in
+        /// <see cref="Zone"/>
```

Comment:
We don't have the same problem here than we do with `GcePlatformDetails` because `Location` already existed and was being filled with the `<zone_name>` part of `Zone`.